participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/config/AppConfig.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Updated.